src/main/java/com/exalttech/trex/ui/views/logs/ConsoleLogView.java

```diff
@@ -21,25 +21,32 @@
 package com.exalttech.trex.ui.views.logs;
 
 import com.exalttech.trex.util.Util;
-import java.util.Date;
 import javafx.application.Platform;
 import javafx.scene.control.TextArea;
 import javafx.scene.input.Clipboard;
 import javafx.scene.input.ClipboardContent;
 import javafx.scene.layout.AnchorPane;
-import static javafx.scene.layout.AnchorPane.setBottomAnchor;
-import static javafx.scene.layout.AnchorPane.setLeftAnchor;
-import static javafx.scene.layout.AnchorPane.setRightAnchor;
-import static javafx.scene.layout.AnchorPane.setTopAnchor;
+
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 /**
  *
  * @author Georgekh
  */
 public class ConsoleLogView extends AnchorPane {
 
-    TextArea logsContent;
-    StringBuffer sb;
+    private TextArea logsContent;
+    private MessageQueue queue = new MessageQueue(5);
+    private Consumer<Void> logUpdater = (e) -> Platform.runLater(() -> {
```

Comment:
Initially I did it through property. And thought it might be useful. Ok, can simplify it.